includes/blocks/class-sensei-course-outline-module-block.php

```diff
@@ -155,9 +155,15 @@ class="wp-block-sensei-lms-course-outline-module__progress-indicator ' . $indica
 	private function get_block_html_attributes( $class_name, $block_attributes ) : string {
 		$class_names   = [ 'wp-block-sensei-lms-course-outline-module', 'sensei-collapsible', $class_name ];
 		$inline_styles = [];
+		$css           = Sensei_Block_Helpers::build_styles(
+			$block_attributes,
+			[
+				'textColor' => null,
+			]
+		);
 
 		if ( ! empty( $block_attributes['bordered'] ) ) {
-			$class_names[] = 'sensei-module-bordered';
+			$class_names[] = 'wp-block-sensei-lms-course-outline-module-bordered';
 
 			if ( ! empty( $block_attributes['borderColorValue'] ) ) {
```

Comment:
@gkaragia, I noticed we're using here a custom logic for the borders, where we use the color even if the user is selecting a color from the palette. Is that for some reason? If yes, maybe this reason is also affecting this PR?

Reply:
The problem with that was that name color classes wouldn't work for borders. So even the appropriate class was applied, the border would still look black.

Reply:
Hum, interesting! Maybe we could try re-visit it some time because it worked for the default border class I applied in this PR. 🤷‍♂️

Reply:
Just adding a note for posterity. I realized now that only the Twenty Twenty contains that class, and only for the `accent` color.